src/xERC20/contracts/XUltraLRT.sol

```diff
@@ -576,4 +617,31 @@ contract XUltraLRT is
     function unpause() public onlyHarvester {
         _unpause();
     }
+
+    ///////////////////////////////////////////////////////////////////////////
+    ////////////////////         TRANSFER LIMITS         //////////////////////
+    ///////////////////////////////////////////////////////////////////////////
+
+    modifier onlyHarvesterOrLockBox() {
+        if (!hasRole(HARVESTER, msg.sender) && msg.sender != lockbox) revert XErrors.NotHarvesterOrLockbox();
+        _;
+    }
+
+    function increaseCrossChainTransferLimit(uint256 _limitInc) public onlyHarvesterOrLockBox {
+        _increaseCrossChainTransferLimit(_limitInc);
+    }
+
+    function decreaseCrossChainTransferLimit(uint256 _limitDec) public onlyHarvesterOrLockBox {
+        _decreaseCrossChainTransferLimit(_limitDec);
+    }
+
+    function _increaseCrossChainTransferLimit(uint256 _limitInc) internal {
+        if ((crossChainTransferLimit + _limitInc) > totalSupply()) revert XErrors.InvalidTransferLimit();
```

Comment:
wait is this true? oh I guess after mint we call this so the total supply has increased, right?

Reply:
You should't increase limit over supply.